src/custom-components/federated-search/components/FedCard.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+    <VsCard
+        class="vs-fed-card"
+        card-style="elevated"
+    >
+        <template #vs-card-header>
+            <div class="vs-fed-card--header">
```

Comment:
I think the BEM class naming is incorrect here. 

It should be "vs-fed-card__header" and anything with "--" is just a modifier. Then it would just be "vs-fed-card__attribute" (you don't need to nest all the classes as header is an element not a block).